File: output/csharp/Search/Search/Profile/QueryBreakdown.cs
using Nest.Internal;
using System;
using System.Collections.Generic;
using System.Runtime.Serialization;
namespace Nest.Search {

	public class QueryBreakdown  {
		
		[DataMember(Name="advance")]
		public long Advance { get; set; }


		[DataMember(Name="build_scorer")]
		public long BuildScorer { get; set; }


		[DataMember(Name="create_weight")]
		public long CreateWeight { get; set; }


		[DataMember(Name="match")]
		public long Match { get; set; }


		[DataMember(Name="next_doc")]
		public long NextDoc { get; set; }


		[DataMember(Name="score")]
		public long Score { get; set; }

	}
}

File: specification/specs/search/search/profile/QueryBreakdown.ts
/*
 * Licensed to Elasticsearch B.V. under one or more contributor
 * license agreements. See the NOTICE file distributed with
 * this work for additional information regarding copyright
 * ownership. Elasticsearch B.V. licenses this file to you under
 * the Apache License, Version 2.0 (the "License"); you may
 * not use this file except in compliance with the License.
 * You may obtain a copy of the License at
 *
 *    http://www.apache.org/licenses/LICENSE-2.0
 *
 * Unless required by applicable law or agreed to in writing,
 * software distributed under the License is distributed on an
 * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
 * KIND, either express or implied.  See the License for the
 * specific language governing permissions and limitations
 * under the License.
 */

class QueryBreakdown {
  advance: long
  advance_count: long
  build_scorer: long
  build_scorer_count: long
  create_weight: long
  create_weight_count: long
  match: long
  match_count: long
  shallow_advance: long
  shallow_advance_count: long
  next_doc: long
  next_doc_count: long
  score: long
  score_count: long
  compute_max_score: long
  compute_max_score_count: long
  set_min_competitive_score: long
  set_min_competitive_score_count: long
}
